portfolio/src/main/webapp/script.js

```diff
@@ -13,60 +13,60 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Add typing animation effect
  */
-var TxtRotate = function(el, toRotate, period) {
-  this.toRotate = toRotate;
-  this.el = el;
-  this.loopNum = 0;
-  this.period = parseInt(period, 10) || 2000;
-  this.txt = '';
-  this.tick();
-  this.isDeleting = false;
-};
+class TxtRotate {
+    constructor(el, toRotate, period) {
+    this.toRotate = toRotate;
+    this.el = el;
+    this.loopNum = 0;
+    this.period = parseInt(period, 10) || 2000;
+    this.txt = '';
+    this.isDeleting = false;
+    this.delta =300 - Math.random() * 100;
+    }
 
-TxtRotate.prototype.tick = function() {
-  let i = this.loopNum % this.toRotate.length;
-  let fullTxt = this.toRotate[i];
+    tick() {
+        let i = this.loopNum % this.toRotate.length;
```

Comment:
I don't see where this i is used, if I didn't overlook anything, you can remove this line.